client/app/models/step.coffee

```diff
@@ -0,0 +1,17 @@
+Backbone = require 'backbone'
+
+# StepModel
+# Backbone wrapper for Onboarding's Step models
+# This Class is a proxy between Backone's logic and native Onboarding Steps
+module.exports = class StepModel extends Backbone.Model
+
+    # Map needed property to current model
+    initialize: ({step}) ->
```

Comment:
I am thinking about renaming Step to OnboardingStep in the future, to clarify. Let's see that after the merge of this PR and the previous one. I am interested by your opinion about this point @CPatchane and @m4dz. :)